tests/tfm/tfm_psa_test/CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ ExternalProject_Add(tfm_psa_arch_test_app
                 -DCROSS_COMPILE=${TFM_TOOLCHAIN_PATH}/${TFM_TOOLCHAIN_PREFIX}
                 -DPSA_TOOLCHAIN_FILE=${TFM_BINARY_DIR}/api_ns/cmake/${TFM_TOOLCHAIN_NS_FILE}
                 -DQCBOR_PATH${QCBOR_PATH_TYPE}=${CONFIG_TFM_QCBOR_PATH}
-                -DCMAKE_BUILD_TYPE=RelWithDebInfo
+                -DCMAKE_BUILD_TYPE=Debug
```

Comment:
So we are always building as Debug, maybe that should be linked to the Kconfig `CONFIG_DEBUG_OPTIMIZATIONS`, if that can be used here?

Reply:
done